### ISSUE TITLE
NVDA doesn not read the posts in facebook when I navigate with j and k keys on microsoft edge

### ISSUE THREAD

**Issue by Fatihyuksek** [nvaccess/nvda#8349]
### Steps to reproduce:
1. Open Edge
2. Browse to www.facebook.com
3. Login
4. Notice you are not able to navigate posts using the J and K keys

### Expected behavior:
You are able to navigate the posts using the J and K keys.

### Actual behavior:
NVDA does not read the posts

### System configuration:
NVDA version:
> Example: next-14027,c80e529f

NVDA Installed or portable:

Other information:
> Example: Running in a VM

Windows version:
> Example: Windows 10 Version 1607 Build 14393.1066

Name and version of other software in use when reproducing the issue:

### Other questions:

Does the issue still occur after restarting your PC?
Yes

Have you tried any other versions of NVDA?
No

**Comment by Fatihyuksek** [comment]
Hello,
When I navigated to the news feed region and turned on focus mode, I pressed j and k and everything was working fine. Thanks.

Windows 10 için Posta<https://go.microsoft.com/fwlink/?LinkId=550986> ile gönderildi

________________________________
Gönderen: Ethan Holliger <notifications@github.com>
Gönderildi: Thursday, May 31, 2018 11:23:30 PM
Kime: nvaccess/nvda
Bilgi: Fatihyuksek; Mention
Konu: Re: [nvaccess/nvda] NVDA doesn not read the posts in facebook when I navigate with j and k keys on microsoft edge (#8349)


@Fatihyuksek<https://github.com/Fatihyuksek>, I have edited your original post to align with the template. A few questions:

  1.  Have you navigated to the "News Feed" region prior to using the J and K keys to navigate posts?
  2.  Are you in focus mode when pressing these keys? You can toggle browse and focus mode manually by pressing NVDA+Space.

**Comment by ehollig** [comment]
@Fatihyuksek, good to hear. Closing as worksforme
